msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,25 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+Here's how to remotely debug your app running in Office app for Android.
+
+   1. From your Android device, launch the Office app.
+   1. Open your profile **Me > Settings > Allow debugging**, and toggle on the option for **Enable remote debugging**.
+:::image type="content" source="images/office-android-enable-remote-debugging.png" alt-text="Screenshot showing Enable remote debugging":::
```

Comment:
Add a line break before the image (otherwise text is inline with image and [looks wonky](https://review.learn.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?branch=pr-en-us-6935&tabs=manifest-teams-toolkit#mobile-debugging))